sa/storage-authority.go

```diff
@@ -637,6 +637,22 @@ func (ssa *SQLStorageAuthority) AlreadyDeniedCSR(names []string) (already bool,
 	return
 }
 
+// CountCertificatesRange returns the number of certificates issued in a specific
+// date range
+func (ssa *SQLStorageAuthority) CountCertificatesRange(start, end time.Time) (count int64, err error) {
+	err = ssa.dbMap.SelectOne(
+		&count,
+		`SELECT COUNT(serial) FROM certificates
```

Comment:
If we want to get pedantic here, `COUNT(1)` might be better. `COUNT(serial)` will give a count of how many certificates have a non-NULL serial field. I believe we have a `NOT NULL` constraint so it shouldn't matter, but it might be a tiny bit more efficient to not have to check the fields. On the other hand, perhaps the MySQL query optimizer can figure out the `NOT NULL` constraint and decide not to check the fields anyhow.